### ISSUE TITLE
Adding element has no effect on Panel

### ISSUE THREAD

**Issue by emerikaji** [icza/gowut#57]
After creating a panel and setting it to *LayoutNatural*, I wanted to add components inside of it. However, I ran into an issue when adding the elements. The first one, which is another panel, got added correctly. The second one however, a link, seems to get skipped when the page is rendered. The page structure goes something like this:
```go
window := gwu.NewWindow("window", "title")
window.SetLayout(gwu.LayoutNatural)

windowPanel := gwu.NewPanel()
windowPanel.SetLayout(gwu.LayoutNatural)

mainPanel := gwu.NewPanel()
mainPanel.SetLayout(gwu.LayoutNatural)
// add things into mainPanel
windowPanel.Add(mainPanel)

confirm := gwu.NewLink("confirm", "url")
windowPanel.Add(confirm)

window.Add(windowPanel)

gowutServer := gwu.NewServer("test", "localhost:8081")
gowutServer.SetText("Test app")
gowutServer.AddWin(window)
gowutServer.Start("window")
```

The affected panel is `windowPanel`. The `confirm` link that should get added doesn't appear on the page nor on the html itself. However, adding a link element manually where it should be using dev tools works as expected.

Is there something wrong with how I add the link to the panel, or is this a bug?
Thank you.

**Comment by icza** [comment]
I tried the exact code you posted above, and I can see the `confirm` link on the page. Probably the things you don't show cause the problem (the `// add things into mainPanel` part...).

**Comment by nishyda** [comment]
```go
```

**Comment by emerikaji** [comment]
I had left this part out since it's the lengthiest part of the code. To sum it up, all I do is create two panels inside the `mainPanel` and add some elements to them.
I've tried removing all the styling from all the components to make sure it wasn't the problem, and yet the link didn't appear; I also tried putting it in a panel of its own, to no effect.

I put the full code in a gist in the event you would want to look at it, and maybe find the problem:
https://gist.github.com/nishyda/d92e5e8f6e0de55474dbb02a834ff8cb
Although, please don't feel obligated to do so, since it's a fair bit lengthier. I will be trying out other solutions.

**Comment by icza** [comment]
Didn't find the mistake at first glance. Try commenting out most part in your code until you see the `confirm` link, then try re-enabling them to see when it "breaks".

**Comment by emerikaji** [comment]
I've finally spotted the issue. In the custom HTML I added to have a PDF viewer, I wrote:
```go
PDFEmbed := gwu.NewHTML("<iframe src=\"/PDF\" type=\"application/pdf\" style=\"height:100%;width:100%;border:none\"/>")
```
However, what was needed was:
```go
PDFEmbed := gwu.NewHTML("<iframe src=\"/PDF\" type=\"application/pdf\" style=\"height:100%;width:100%;border:none\"></iframe>")
```
It seems for some reason closing the tag directly instead of writing the tag name again to close it made Gowut interpret it weirdly, and not add the elements after it.
Is it because when Gowut was written, closing tags directly (like `<input />` instead of `<input></input>`) was not supported yet?
